apps/fabric-website/index.html

```diff
@@ -156,7 +156,7 @@
     <script type="text/javascript">
       // @ts-check
       var isLocal = location.hostname === 'localhost' || location.hostname.indexOf('ngrok.io') > -1;
-      var isDev = isLocal || getParameterByName('isDev') === '1';
+      var isDev = isLocal || getParameterByName('dev') === '1' || !!getParameterByName('strict');
```

Comment:
where is `var isDev` being used?

Reply:
also, looks odd to have both `isDev` and `isProduction`.

Reply:
Oops, there's some logic I forgot to copy from homepage.htm.

Reply:
`isDev` determines whether to use dev mode (non-minified scripts and dev mode react). `isProduction` determines whether to use the production manifest or dogfood/staging manifest.